shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -329,6 +329,7 @@ - (bool)engineCallbackOnClearCurrent {
 
 - (bool)engineCallbackOnPresent {
   if (!_mainOpenGLContext) {
+    return false;
```

Comment:
I am surprised `-[NSOpenGLContext flushBuffer]` returns void. Otherwise, we could have `return [_mainOpenGLContext flushBuffer]`. Oh well.